src/components/NewsletterForm.tsx

```
@@ -0,0 +1,186 @@
+        <h3 className="text-lg font-bold text-green-800 mb-2">Thank You!</h3>
+        <p className="text-green-700">
+          {`You've been signed up for our newsletter. We'll send you updates on our latest content.`}
+        </p>
+        <button 
+          onClick={() => {
+            setSubmitted(false);
+            setFormData({ email: '', firstName: '' });
+            setErrors({});
+          }}
+          className="mt-4 px-4 py-2 bg-green-600 text-white rounded hover:bg-green-700 text-sm"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)